VideoEncoding/Encoding_H264/Program.cs

```diff
@@ -1,721 +1,523 @@
+    // Does a Transform already exist with the desired name? This method will just overwrite (Update) the Transform if it exists already. 
+    // In production code, you may want to be cautious about that. It really depends on your scenario.
+    var transform = await mediaServicesAccount.GetMediaTransforms().CreateOrUpdateAsync(
+        WaitUntil.Completed,
+        transformName,
+        new MediaTransformData
         {
-
-            Console.WriteLine("Creating a custom transform...");
-
-            // Create a new Transform Outputs array - this defines the set of outputs for the Transform
-            TransformOutput[] outputs = new TransformOutput[]
+            Outputs =
             {
-                    // Create a new TransformOutput with a custom Standard Encoder Preset
-                    // This demonstrates how to create custom codec and layer output settings
-
-                  new TransformOutput(
-                        new StandardEncoderPreset(
-                            codecs: new Codec[]
+                // Create a new TransformOutput with a custom Standard Encoder Preset using the HEVC (H265Layer) codec
+                // This demonstrates how to create custom codec and layer output settings
+                new MediaTransformOutput(
+                    preset: new StandardEncoderPreset(
+                        codecs: new MediaCodecBase[]
+                        {
+                            // Add an AAC Audio layer for the audio encoding
+                            new AacAudio
                             {
-                                // Add an AAC Audio layer for the audio encoding
-                                new AacAudio(
-                                    channels: 2,
-                                    samplingRate: 48000,
-                                    bitrate: 128000,
-                                    profile: AacAudioProfile.AacLc
-                                ),
-                                // Next, add a H264Video for the video encoding
-                               new H264Video (
-                                    // Set the GOP interval to 2 seconds for all H264Layers
-                                    keyFrameInterval:TimeSpan.FromSeconds(2),
-                                     // Add H264Layers. Assign a label that you can use for the output filename
-                                    layers:  new H264Layer[]
+                                Channels = 2,
+                                SamplingRate = 48000,
+                                Bitrate = 128000,
+                                Profile = AacAudioProfile.AacLc
+                            },
+                            // Next, add a H.264 for the video encoding
+                            new H264Video
+                            {
+                                // Set the GOP interval to 2 seconds for all H.265 layers
```

Comment:
H.265 is referenced but it's H.264 here

Reply:
I've updated the code to resolve this issue.